extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -62,6 +63,15 @@ export class PublishOptionsDialog {
 
 	private initializeDeploymentOptionsDialogTab(): void {
 		this.optionsTab?.registerContent(async view => {
+			// creating loader component
+			this.loader = view.modelBuilder.loadingComponent()
```

Comment:
loader doesn't have to be kept as a variable of the class if it's only used in this function